src/persistence/settings.cpp

```diff
@@ -492,7 +492,7 @@ void Settings::savePersonal(Profile* profile)
     savePersonal(profile->getName(), profile->getPassword());
 }
 
-void Settings::savePersonal(QString profileName, QString password)
+void Settings::savePersonal(QString profileName, const QString &password)
```

Comment:
So, how it's better to set up workflow? Should i add new commits as part of this pr, or this can be merged, and new one go as separate pr?